upgrade/patches/41_custom_field_required.sql

```diff
@@ -0,0 +1,3 @@
+-- Add field to allow custom fields to be marked as required when editing
+alter table {{%custom_field}}
+    add `fld_edit_form_required` tinyint(1) NOT NULL DEFAULT 0;
```

Comment:
please add `AFTER` statement here, so fields in table would be ordered like:
 - primary key
 - int values
 - text values

this supposedly improves key lookup performance.
i'd like to follow this idea in eventum

also probably should also order int fields too, but not sure in what order, smallest size first or biggest size first. or doesn't matter, just be consistent?